addons/mail/static/src/core/common/message_reaction_menu.js

```diff
@@ -14,7 +14,7 @@ import { Dialog } from "@web/core/dialog/dialog";
 import { useService } from "@web/core/utils/hooks";
 
 export class MessageReactionMenu extends Component {
-    static props = ["close", "message"];
+    static props = ["close", "message", "currentReaction?"];
```

Comment:
Nitpicking: I renamed to `initialReaction`, so it's clearer what this props mean. "current" is too broad when we don't see how it's actually used imperatively. Props are declarative so they should be named as such.